server/server/src/metrics.rs

```diff
@@ -152,7 +161,7 @@ pub async fn get_user_billing_platform(
 
 pub async fn get_user_info(
     state: &ServerState, owner: Owner,
-) -> Result<(i64, i64, bool), ServerError<MetricsError>> {
+) -> Result<(i64, i64, bool, bool), ServerError<MetricsError>> {
```

Comment:
this crosses the threshold for me with regards to number of acceptable tuple variants, I would make this a struct at this point. IMO as soon as there were 2 different fields with the same type we should have done it then.